src/main/scala/com/tapad/docker/DockerComposePlugin.scala

```diff
@@ -252,7 +257,7 @@ class DockerComposePluginLocal extends AutoPlugin with ComposeFile with DockerCo
       // If the compose file being used is a version that creates a new network on startup then remove that network on
       // shutdown
       if (new File(composePath).exists()) {
-        val composeYaml = readComposeFile(composePath)
+        val composeYaml = readComposeFile(composePath, variables)
```

Comment:
Since this call to `readComposeFile` only is to get the Version information from the compose file it doesn't need to actually substitute any variables. (I think its actually using the already variable substituted version of the compose file in this call already) I was thinking that in this case the `variables` parameter should not be passed and the default `Vector.empty` value will be used. You just need to make sure that the code in `readComposeFile` will gracefully skip the substitution when `Vector.empty` is used.